article/test/models/LiveBlogCurrentPageTest.scala

```diff
@@ -1,4 +1,4 @@
-package liveblog
+package models
```

Comment:
I think this was introduced by the IntelliJ refactor (which sucks) without me noticing. I thought I'd fixed all of the problems but this one was dangling